vm/metrics.go

```diff
@@ -66,71 +66,63 @@ func newMetrics() (*prometheus.Registry, *Metrics, error) {
 	r := prometheus.NewRegistry()
 
 	rootCalculated, err := metric.NewAverager(
-		"chain",
-		"root_calculated",
+		"chain_root_calculated",
```

Comment:
Here and in pebble I simply added the namespace as a prefix to the name of a metric to accommodate the change in AvalancheGo, which removes the parameter for specifying a namespace.

I'd prefer the namespace parameter was kept in AvalancheGo, but this seems like the best way to reproduce the same metrics for this update PR.

Reply:
Good catch. Was not aware of this change.

Reply:
> Wondering if there is a new way to replicate this behavior or if it is entirely removed?

It's entirely removed. The current change is the way to implement this behavior.